src/toolbar.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { startHighlight, endHighlight } from './highlight';
+import Tooltip from '@material-ui/core/Tooltip'
+import Button from '@material-ui/core/Button';
+import ToggleButton from '@material-ui/lab/ToggleButton';
+
+const toolbarStyle: React.HTMLAttributes<HTMLDivElement>["style"] = {
+  display: 'flex',
+  flexDirection: 'column',
+  position: 'fixed',
+  top: '25%',
+  right: '0px',
+  padding: '1px'
+};
+
+const toolbarContentStyle: React.HTMLAttributes<HTMLDivElement>["style"] = {
+  display: 'flex',
+  backgroundColor: '#fafafa',
+  flexDirection: 'column',
+  right: '0px',
+  border: '1px solid #757575'
+};
+
+const toolbarToggleStyle: React.HTMLAttributes<HTMLDivElement>["style"] = {
+  backgroundColor: '#fafafa',
+  width: '30px',
+  right: '0px',
+  padding: '5px'
+};
+
+const toolbarIconStyle: React.HTMLAttributes<HTMLDivElement>["style"] = {
+  opacity: '0.5',
+  width: '45px',
+  height: '45px'
+};
+
+const toolbarButtonStyle: React.HTMLAttributes<HTMLDivElement>["style"] = {
+  borderStyle: "none",
+  color: "black",
+  backgroundColor: "#fafafa"
+};
+
+const toolbarToggleableStyle: React.HTMLAttributes<HTMLDivElement>["style"] = {
+  borderStyle: "none"
+};
+
+
+const Toolbar = () => {
+
+  const [isActive, changeIsActive] = React.useState(false);
+
+  const handleToggle = () => {
+    changeIsActive(!isActive);
+  };
```

Comment:
In this case, please use const [isActive, handleToggle] = React.useReducer(a => !a, false);